### DISCUSSION TITLE
How Make a new branch in a Repo from GitHubCodeSpace

### DISCUSSION THREAD

**Discussion by wilonweb** [community/community#40851] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hello 
I did a repo but i don't know how add a new branch in the repo. 

To do that i tried these steps 

I opened the repo with GitHub CodeSpace 
In the gitbash i enter these commands
```
gitbranch ok 
gitcheckout ok 
git add .
git commit -m "test"
git push origin -u main
```
But when i go on my Repo i don't see the branch "OK" 
Do you know why ? 

https://github.com/NinjaTurtles-cloud/Create-React-App2/tree/main

**Comment by wilonweb** [comment, marked as answer]
Is Okay !!! 
I had to use git bash ^^

**Comment by mohamedzhioua** [comment]
good
